lib/Command/ExApp/Update.php

```diff
@@ -50,8 +50,10 @@ protected function configure(): void {
 		$this->addOption('force-scopes', null, InputOption::VALUE_NONE, 'Force new ExApp scopes approval[deprecated]');
 		$this->addOption('wait-finish', null, InputOption::VALUE_NONE, 'Wait until finish');
 		$this->addOption('silent', null, InputOption::VALUE_NONE, 'Do not print to console');
-		$this->addOption('all', null, InputOption::VALUE_NONE, 'Update all updatable apps');
+		$this->addOption('all', null, InputOption::VALUE_NONE, 'Update all enabled and updatable apps');
 		$this->addOption('showonly', null, InputOption::VALUE_NONE, 'Additional flag for "--all" to only show all updatable apps');
+		$this->addOption('include-disabled', null, InputOption::VALUE_NONE, 'Update also disabled apps (use --enable-after-update to enable ExApp after update)');
+		$this->addOption('enable-after-update', null, InputOption::VALUE_NONE, 'Enable ExApp after update if it was disabled before. Use with --include-disabled');
```

Comment:
Enable**s** **an** ExApp after **the** update if it was disabled before. Use with --include-disabled

Question: 

    What is the purpose of this option? When might it be useful?
    Perhaps it shouldn't be added, to avoid complicating the logic too much?

*If the decision to add it is correct, it should be documented clearly, explaining when, why, and for what it can be used.*

Reply:
Right, this is redundant, removed.